pkg/gitreceive/k8s_util.go

```diff
@@ -12,10 +12,11 @@ import (
 )
 
 const (
-	slugBuilderName    = "deis-slugbuilder"
-	slugBuilderImage   = "quay.io/deisci/slugbuilder:v2-beta"
-	dockerBuilderName  = "deis-dockerbuilder"
-	dockerBuilderImage = "quay.io/deisci/dockerbuilder:v2-beta"
+	slugBuilderName   = "deis-slugbuilder"
+	slugBuilderImage  = "quay.io/deisci/slugbuilder:v2-beta"
+	dockerBuilderName = "deis-dockerbuilder"
+	// change the value back to quay.io/deisci/dockerbuilder:v2-beta
+	dockerBuilderImage = "smothiki/dockerbuilder:v1.8"
```

Comment:
Note to readers: this needs to be changed when https://github.com/deis/dockerbuilder/issues/4 is complete. Tracked in #142. Not blocking for this PR